hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -131,90 +131,90 @@ struct Vunit;
   O(nop, Inone, Un, Dn)\
   O(ud2, Inone, Un, Dn)\
   /* arithmetic instructions */\
```

Comment:
Vinstr has an annotated two-byte hole.  Why not put it there?  It also avoids the churn of adding flag bits to all the sub-structs, which is nice since it's not used at all on x64 (or on ppc64, at least yet).